ebau_gwr/core/hs_client.py

```diff
@@ -47,8 +72,16 @@ def get(self, path="constructionprojects", query=None, headers=None):
         url = "/".join([self.base_uri, path])
         response = self._request("get", url, headers=headers)
 
-        # TODO: the housing stat api does not support version 2 yet
-        content = response.content.replace(b"eCH-0216/1", b"eCH-0216/2")
+        # TODO: the housing stat api does not deliver valid XML yet
+        content = self._fix_housing_stat_response(response.content)
 
-        data = CreateFromDocument(content)
+        try:
+            data = CreateFromDocument(content)
+        except (
+            IncompleteElementContentError,
+            UnprocessedElementContentError,
+        ) as e:  # pragma: no cover
+            # TODO: correctly report errors back
+            print(e.details())
```

Comment:
Maybe use log instead of print?

Reply:
Done. But it's only for development until we have proper error handling.